src/commonMain/kotlin/teksturepako/pakku/api/projects/ProjectFile.kt

```diff
@@ -27,6 +29,7 @@ data class ProjectFile(
     val hashes: MutableMap<String, String>? = null,
     @SerialName("required_dependencies") val requiredDependencies: MutableSet<String>? = null,
     val size: Int = 0,
+    @SerialName("data_published") val dataPublished: @Serializable(with = InstantIso8601Serializer::class) Instant = Instant.MIN,
```

Comment:
There is a typo: `@SerialName("data_published") val dataPublished`
it should be: `@SerialName("date_published") val datePublished`